Addons/tsgExoticQuadrature.hpp

```diff
@@ -307,20 +307,20 @@ inline void shiftReferenceWeights(std::vector<double> const &vals, double shift,
  * \ingroup TasmanianAddonsExoticQuad
  * \brief Constructs an exotic quadrature from a sparse grid object.
  *
- * Constructs a set of one dimensional quadrature rules with respect to a weights defined in \b grid.
+ * Constructs a set of one dimensional quadrature rules with respect to a 1D weight function approximated by \b grid.
  *
  * \param num_levels is the number of levels that should be computed
- * \param shift is the shift that needs to be added to the weights to make it positive
+ * \param shift is a real-valued scalar that needs to be added to the weight function to make it positive
  * \param grid has dimension 1 and at least one loaded point, should use as many points as possible for better accuracy
  *        (up to some point where numerical stability is lost)
  * \param description is a human readable string that can be used to identify the quadrature rule (could be empty)
- * \param is_symmetric indicates whether to assume that the weight is symmetric which allows for some stability improvements;
- *        symmetry is defined as "even" function on [-1, 1] which means that all odd power polynomials integrate to zero;
+ * \param is_symmetric indicates whether to assume that the weight is symmetric, which allows for some stability improvements;
+ *        symmetry is defined as being an "even" function on [-1, 1], which means that all odd power polynomials integrate to zero;
```

Comment:
Expressions such as `being an` often over-encumber the language more than help streamline the point.

How about:
```
symmetric means "even" on [-1, 1] which leads to all odd power polynomials integrating to zero
```